.travis.yml

```diff
@@ -103,21 +102,23 @@ matrix:
           env: PYTHON_VERSION=3.4 NUMPY_VERSION=1.10
         - os: linux
           env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.11
+        - os: linux
+          env: NUMPY_VERSION=1.12
 
         # Try numpy pre-release
         - os: linux
           env: NUMPY_VERSION=prerelease
                EVENT_TYPE='pull_request push cron'
 
-#        # Do a PEP8 test with pycodestyle
-#        - os: linux
-#          env: MAIN_CMD='pycodestyle specviz --count' SETUP_CMD=''
+        # Do a PEP8 test with pycodestyle
+        - os: linux
+          env: MAIN_CMD='pycodestyle packagename --count' SETUP_CMD=''
 
     allow_failures:
         # Do a PEP8 test with pycodestyle
         # (allow to fail unless your code completely compliant)
         - os: linux
-          env: MAIN_CMD='pycodestyle specviz --count' SETUP_CMD=''
+          env: MAIN_CMD='pycodestyle packagename --count' SETUP_CMD=''
```

Comment:
Here and above, looks like this needs to be changed back to specviz

Reply:
Oops, good catch.